libs/vm-modules/tests/unit/big_number_wrapper_tests.cpp

```diff
@@ -306,46 +451,42 @@ TEST_F(UInt256Tests, uint256_logValue)
   EXPECT_NEAR(result, expected, expected * LOGARITHM_TOLERANCE);
 }
 
+// Warning: the test contains 1 temporary disabled assertion.
```

Comment:
ditch the comment - handle the tests correctly

Reply:
done